exonum-java-binding/core/src/test/java/com/exonum/binding/core/runtime/ServiceRuntimeIntegrationTest.java

```diff
@@ -257,7 +257,7 @@ void commitService() {
         .thenReturn(serviceWrapper);
 
     // Create the service from the artifact
-    serviceRuntime.commitService(instanceSpec);
+    serviceRuntime.updateInstanceStatus(instanceSpec, null);
```

Comment:
What is expected to be in place for `null`? An enum value? If that's the case, I think it would be more appropriate than `null`, even if it is unused.